components/style/properties/properties.mako.rs

```diff
@@ -1552,6 +1552,9 @@ impl PropertyDeclaration {
         assert!(declarations.is_empty());
         match id {
             PropertyId::Custom(name) => {
+                // FIXME: fully implement https://github.com/w3c/csswg-drafts/issues/774
+                // before adding skip_whitespace here.
+                // This probably affects some test results.
```

Comment:
This would also imply we should start adding spaces after the colon for a custom property declaration, perhaps worth leaving a note near the relevant code in `declaration_block.rs`.